Amazon.QLDB.Driver/transaction/TransactionExecutor.cs

```diff
@@ -66,6 +70,19 @@ public IResult Execute(string statement)
         {
             return this.transaction.Execute(statement);
         }
+        
+        /// <summary>
+        /// Execute the IQuery against QLDB and retrieve the result.
+        /// </summary>
+        public Amazon.QLDB.Driver.Generic.IResult<T> Execute<T>(IQuery<T> query)
+        {
+            return this.transaction.Execute(query);
+        }
+
+        public IQuery<T> Query<T>(string statement, params object[] parameters)
```

Comment:
Need docs here - if this method is being kept and the suggestion for async isn't used.

Reply:
Added docstrings